### PR TITLE
Update to Polymer 1.5

### PR DESCRIPTION
**Pull request by freakypie**

### CODE REVIEW

**Comment by ruyadorno** [comment]
@freakypie thank you so much for taking the time to do this so much needed update

😊 I'll review it ASAP

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymer-simple-slider",
-  "version": "0.2.0",
+  "version": "0.3.0",
```
Comment (avdaredevil) [inline bower.json]: @freakypie All libs on Polymer v1+ normally have a major version bump (ie. `1.0.0`)
Food for thought

cc: @ruyadorno Could you expedite this pull request??

**Comment by ruyadorno** [comment]
👍 manually merged into master, landed on **v1.0.0**

let me know if you find any issues!

many thanks @freakypie and @avdaredevil 🎉

File: simple-slider.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          slider.resumeAutoPlay();
+          console.log("resuming");
+        });
+      }, this.transitionStartDelay || 0);
+      this.slider = slider;
+    },
+
+    // Dispose slider on element detach
+    detached: function () {
+      element.dispose();
```
Comment (ruyadorno) [inline simple-slider.html]: worry not, this was fixed in a _post-merge_ commit on master, **v1.0.0** contains the fix 😄

**Pull request closed without merging**